benchexec/tools/brick.py

```diff
@@ -10,22 +10,28 @@
 import benchexec.result as result
 
 
-class Tool(benchexec.tools.template.BaseTool):
+class Tool(benchexec.tools.template.BaseTool2):
     """
     Tool info for BRICK
     https://github.com/brick-tool-dev/brick-tool
     """
 
     REQUIRED_PATHS = ["bin", "lib"]
 
-    def executable(self):
-        return util.find_executable("bin/brick")
+    def executable(self, tool_locator):
+        return tool_locator.find_executable("brick", subdir="bin")
 
     def name(self):
         return "BRICK"
 
-    def cmdline(self, executable, options, tasks, propertyfile, rlimits):
-        return [executable] + options + tasks
+    def cmdline(self, executable, options, task, rlimits):
+        data_model_param = util.get_data_model_from_task(
+            task, {"ILP32": "--32", "LP64": "--64"}
```

Comment:
I think we should wait for https://github.com/sosy-lab/sv-benchmarks/issues/1125 to be resolved before handling this.

Reply:
@PhilippWendler now I wonder if we should wait for https://github.com/sosy-lab/sv-benchmarks/issues/1125 to be resolved, or just go ahead with this and incorporate the changes later.

Reply:
I would say that refactoring the constants out into the utility module should be done, because it might make adjusting the tool-info modules to the result of sosy-lab/sv-benchmarks#1125 easier.

But we could avoid blocking this PR on sosy-lab/sv-benchmarks#1125 if there is no progress there.